tests/extension/oneapi_kernel_compiler_spirv/kernel_compiler_spirv.cpp

```diff
@@ -116,7 +116,7 @@ void testParam(sycl::queue& q, const sycl::kernel& kernel) {
     cgh.set_arg(1, b_ptr);
     // Pass sycl::accessor for OpTypePointer(CrossWorkgroup) parameter.
     cgh.set_arg(
-        2, output_buffer.template get_access<sycl::access::mode::write>(cgh));
+        2, sycl::accessor<T, 1, sycl::access_mode::write>{output_buffer, cgh});
```

Comment:
Sure, but the idea is to use CTAD + tag dispatching
```suggestion
        2, sycl::accessor{ output_buffer, cgh, sycl::write_only });
```